src/app/app.module.ts

```diff
@@ -26,11 +28,21 @@ import { PageNotFoundComponent } from './page-not-found/page-not-found.component
     PageNotFoundComponent
   ],
   imports: [
+    NgxAuthFirebaseUIModule.forRoot({
```

Comment:
I think even if its safe to expose these values in UI code, we should have a separate configuration file to manage this.

Reply:
Added a task to address this in backlog.